tools/atip/tools/webdriver.xw_tizen.json

```diff
@@ -11,5 +11,5 @@
         "device": "", 
         "name": "xw_tizen"
     }, 
-    "url-prefix": "file:///opt/home/app/.config/xwalk-service/applications/TEST_APP_ID/"
```

Comment:
Just add `/` here as a workaround.

Since I have done the replacement of `TESTER-HOME-DIR` for `/home/app`, I will merge this pull request as-is. Feel free to update them all if you insist on `home/app`.